examples/metatensor-plumed/lj-crystal.py

```diff
@@ -0,0 +1,543 @@
+# -*- coding: utf-8 -*-
+"""
+Exploring the Lennard-Jones 38 Cluster with Metadynamics
+=========================================================
+
+:Authors: Guillaume Fraux `@Luthaf <https://github.com/luthaf/>`_;
+          Rohit Goswami `@HaoZeke <https://github.com/haozeke/>`_;
+          Michele Ceriotti `@ceriottim <https://github.com/ceriottim/>`_
+
+We shall demonstrate the usage of the metatensor ecosystem within enhanced
+sampling techniques, specifically by running **metadynamics**, to explore the
+complex potential energy surface (PES) of a 38-atom Lennard-Jones (LJ) cluster.
+The LJ38 cluster is a classic benchmark system because its global minimum energy
+structure is a truncated octahedron with :math:`O_h` symmetry, which is
+difficult to find with simple optimization methods. The PES has a multi-funnel
+landscape, meaning the system can easily get trapped in local minima.
+
+Our goal is progressively transition from a random configuration to the low-energy
+structures. To do this, we will:
+
+1.  Define a set of **collective variables (CVs)** that can distinguish between
+    the disordered (liquid-like) and ordered (solid-like) states of the
+    cluster. We will use a custom CV based on **Steinhardt order parameters**
+    (:math:`Q_4` and :math:`Q_6`, a.k.a the bond-order parameters).
+2.  Implement this custom CV using `featomic`, `metatensor`, and `metatomic` to
+    create a portable `metatomic` model.
```

Comment:
```suggestion
2.  Implement this custom CV using ``featomic``, ``metatensor``, and ``metatomic`` to
    create a portable ``metatomic`` model. This serves as an example of using machine learning tools to define CV, and the actual implementation could be switched to anything your heart desire!
```

Or something similar!

Reply:
>.< Ugh sphinx, always forget `` is internal linkage